### ISSUE TITLE
QUploader - createObjectURL issue for img thumbnails

### ISSUE THREAD

**Issue by psta-zeus** [quasarframework/quasar#7127]
**Describe the bug**
When compile a Quasar app using Cordova for Android (iOS too), there is error when I try upload new image using QUploader.

**Codepen/jsFiddle/Codesandbox (required)**
Not applied, its Cordova app

**To Reproduce**

For Android:
Create page with a QUploader component (thumbnails enabled)
Compile the application with Cordova and generate the APK
Install the .apk into Android phone
In app add image file using QUploader
Check console (Failed to execute 'createObjectURL' on 'URL': No function was found that matched the signature provided.)

**Expected behavior**
Update __addFiles method in QUploaderBase - replace createObjectURL with currently supported solution.

**Platform:**
OS: Android
Node: v10.13.0
NPM: 6.14.5
Cordova: 9.0.0
Yarn: - Not used -
Browsers: - No applied -
Android: 8.x.x, 9.x.x,10.x.x
Electron: - No applied

**Comment by rstoenescu** [comment]
createObjectURL is NOT deprecated :)
Do you use the default webview or a custom one (as indicated in the Quasar docs)?

**Comment by rstoenescu** [comment]
What's the format of the image file being supplied to QUploader? (png, jpg, etc)

**Comment by psta-zeus** [comment]
> What's the format of the image file being supplied to QUploader? (png, jpg, etc)

Both jpg, png

**Comment by psta-zeus** [comment]
> createObjectURL is NOT deprecated :)
> Do you use the default webview or a custom one (as indicated in the Quasar docs)?
Sorry, if it's not correct. When I was looking for a solution, the most common result was information that is deprecated and needs to be replaced.

We are using cordova-plugin-ionic-webview 4.1.3

**Comment by pdanpdan** [comment]
Re you sure you're not using cordova-plugin-camera (https://github.com/ionic-team/ionic-native/issues/2868)? I would rather expect the crash to be on selection.

**Comment by psta-zeus** [comment]
> Re you sure you're not using cordova-plugin-camera ([ionic-team/ionic-native#2868](https://github.com/ionic-team/ionic-native/issues/2868))? I would rather expect the crash to be on selection.

Yes, you are right. In app is used cordova-plugin-camera. I was focused on Quasar, because this bug we found after Quasar update (v1.9.1).

We were unable to use QUploader in the same way as in the browser, where the user can choose how to add the file. On mobile there is popup "Select action" with all available options (camera, file storage,..)
We developed custom choose file component. It's used when user click on add btn in QUploader. This component use cordova-plugin-camera or cordova-plugin-file.
